Tasks/WindowsMachineFileCopy/RoboCopyJob.ps1

```diff
@@ -174,6 +174,31 @@ param (
         return 'net'
     }
     
+    function Write-DTLServiceDeprecationMessageIfRequired
+    {
+        param([string]$machine)
+    
+         try
+        {
+            $jsonValue = ConvertFrom-Json $machine -ErrorAction Stop;
+            $validJson = $true;
+        }
+        catch
+        {
+                $validJson = $false;
+        }
+
+        if(!$validJson)
+        {
+            $dtlServiceInValidChars = @('.',':',',')
+
+            if($machine.Contains('.') -or $machine.Contains(':') -or $machine.Contains(","))
```

Comment:
this should 'not' of the condition

Reply:
condition was corrected but missed in porting to M123 branch